tests/kernel/wave/wave_gemm_test.py

```diff
@@ -310,7 +312,7 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
                 )
         iree_ref = torch.zeros(shape[0], shape[1], dtype=torch.float32)
         generate_iree_ref("mmt_f8", [a, b], [iree_ref], config, run_bench=run_bench)
-        assert_close(c, iree_ref)
+        assert_close(c, iree_ref, atol=3e-5, rtol=3e-4)
```

Comment:
Why does this change? Is it the accumulation order? Do you also see this in IREE?

Reply:
yeah, I think it is indeed the acc order, didn't you say the hipblas folks also encountered something similar? Funny enough only `F32_16x16x32_K4_F8` seem to have this slight issue with acc

Reply:
That makes sense, but the hard part here is since our current `generate_iree_ref` is quite high level, is not very easy to select which mma intrinsics we want to use haha

Reply:
We can definitely modify `generate_iree_ref` to do that, and won't take too too long, but it would probably warrant another PR, since this one is getting quite bloated already 😁

Reply:
Sounds good, yes feel free to move that to a subsequent PR.